cognite/client/data_classes/data_modeling/views.py

```diff
@@ -863,3 +866,72 @@ def dump(self, camel_case: bool = True) -> dict[str, Any]:
             output["connection_type"] = "multi_reverse_direct_relation"
 
         return output
+
+
+@dataclass
+class PropertyUnitReference:
```

Comment:
I would suggest this name instead.
```suggestion
class PropertyUnit:
```